test_shop.py

```diff
@@ -7,7 +7,6 @@
 from shop import SellableItem, Shop
 
 
-@pytest.mark.only
```

Comment:
What did this do?

Reply:
That makes it so that when you run the unit tests, ONLY that unit test actually runs. It skips the rest. It should have never been merged. But it's really useful during development.

Reply:
Good to know.